transmogrifier/sources/json/aardvark.py

```diff
@@ -90,9 +90,14 @@ def record_is_deleted(cls, source_record: dict) -> bool:
         Args:
             source_record: A JSON object representing a source record.
         """
-        if source_record.get("gbl_suppressed_b") is True:
-            return True
-        return False
+        if isinstance(source_record["gbl_suppressed_b"], bool):
+            return source_record["gbl_suppressed_b"]
+        else:
+            message = (
+                f"Record ID '{cls.get_source_record_id(source_record)}': "
+                "'gbl_suppressed_b' value is not a boolean"
+            )
+            raise ValueError(message)
```

Comment:
I think this is good.  If we don't know definitively if the record is deleted or not, raises an error.